openedx/core/djangoapps/course_groups/tests/test_cohorts.py

```diff
@@ -685,12 +685,43 @@ def test_add_user_to_cohort(self, mock_tracker):
             lambda: cohorts.add_user_to_cohort(first_cohort, "non_existent_username")
         )
 
+    def test_get_course_cohort_settings(self):
+        """
+        Test that cohorts.get_course_cohort_settings is working as expected.
+        """
+        course = modulestore().get_course(self.toy_course_key)
+        course_cohort_settings = cohorts.get_course_cohort_settings(course.id)
+
+        self.assertFalse(course_cohort_settings.is_cohorted)
+        self.assertEqual(course_cohort_settings.cohorted_discussions, [])
+        self.assertTrue(course_cohort_settings.always_cohort_inline_discussions)
+
+    def test_update_course_cohort_settings(self):
+        """
+        Test that cohorts.set_course_cohort_settings is working as expected.
```

Comment:
It would be good to do some negative testing against this API too. What happens with invalid values? What happens if required fields are not supplied?